core-tests/src/test/java/org/mule/runtime/core/routing/filters/ExpressionFilterTestCase.java

```diff
@@ -137,15 +136,14 @@ public void testRegexFilterWithAngleBrackets() {
 
   @Test
   public void testExceptionTypeFilter() {
-    ExpressionFilter filter = new ExpressionFilter("exception is java.lang.Exception");
+    ExpressionFilter filter = new ExpressionFilter("exception.getCause() is java.lang.Exception");
```

Comment:
Can you add a comment about this in the compatibility section of the error handling document?